gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/internal/model/SubscriptionMongo.java

```diff
@@ -59,6 +59,11 @@ public class SubscriptionMongo extends Auditable {
      */
     private String status;
 
+    /**
+     * ACTIVE, PAUSED
+     */
+    private String consumerStatus;
```

Comment:
This is related to other comments I've let but why not set consumerStatus to `ACTIVE` by default so you avoid always checking for nullability.

Reply:
changed :)